### PR TITLE
[17.03.x] Fix the generated bindata in the release branch

### LINKED ISSUE #30284: internal Additional property internal is not allowed (docker stack deploy with compose v3 file)
**Description**

The `internal` key doesn't seem to work when creating a network with `docker stack deploy` and a docker-compose file (v3 format).

**Steps to reproduce the issue:**
1. create a `docker-compose.yml` file with this content:
```yaml
version: '3.0'

networks:
  foo:
    driver: overlay
    internal: true
```
2. deploy it:
```console
$ docker stack deploy -c docker-compose.yml foo
```
3. watch it fail 💥:
```console
$ docker stack deploy -c docker-compose.yml foo
internal Additional property internal is not allowed
```

**Describe the results you received:**

(see above)

**Describe the results you expected:**

An internal network to be created :)

**Output of `docker version`:**

```
$ docker version
Client:
 Version:      1.13.0-rc7
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   48a9e53
 Built:        Fri Jan 13 21:41:57 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      1.13.0-rc7
 API version:  1.25 (minimum version 1.12)
 Go version:   go1.7.3
 Git commit:   48a9e53
 Built:        Fri Jan 13 21:41:57 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Output of `docker info`:**

```
$ docker info
Containers: 2
 Running: 1
 Paused: 0
 Stopped: 1
Images: 606
Server Version: 1.13.0-rc7
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
Swarm: active
 NodeID: 62ji4rbm9cp6lluyrz7acl00y
 Is Manager: true
 ClusterID: ap5aksi3rjbalc5zvhrut4rf7
 Managers: 1
 Nodes: 1
 Orchestration:
  Task History Retention Limit: 10
 Raft:
  Snapshot Interval: 10000
  Number of Old Snapshots to Retain: 0
  Heartbeat Tick: 1
  Election Tick: 3
 Dispatcher:
  Heartbeat Period: 5 seconds
 CA Configuration:
  Expiry Duration: 3 months
 Node Address: 192.168.65.2
 Manager Addresses:
  192.168.65.2:2377
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 03e5862ec0d8d3b3f750e19fca3ee367e13c090e
runc version: 2f7393a47307a16f8cee44a37b262e8b81021e3e
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.3-moby
Operating System: Alpine Linux v3.5
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 1.952 GiB
Name: moby
ID: 5OHX:44FP:GRVY:CK7Y:J7R5:P7YY:YMZB:JN4G:HN2G:IWIE:L742:SHNP
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 67
 Goroutines: 353
 System Time: 2017-01-19T16:56:58.766066099Z
 EventsListeners: 2
No Proxy: *.local, 169.254/16
Username: hairyhenderson
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Docker for Mac 1.13.0-rc7-beta37 (15013)

### PR DESCRIPTION
**Pull request by dnephin**
Fixes #30284 (for the release branch, it's correct in master).

I'm not sure how this happened but somehow in the cherry-picking the bindata changes were lost.

cc @vieux

### CODE REVIEW

File: cli/compose/schema/bindata.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func (fi bindataFileInfo) Sys() interface{} {
...
...a4\x00\x07\xef\xee\x75\xb4\xd2\x48\x28\xd7\x57\x97\xf3\x7b\xcd\xba\xa3\x9a\x0f\xf9\x19\x41\xfd\x81\x2e\xfe\x26\x35\x80\xf4\x85\xac\xa3\xf3\xa0\x0a\x2a\x81\xde\x04\x5c\xe0\xcd\x77\xcc\xc4\x9e\x6c\x81\xaa\x36\x6b\x80\xd8\x51\x99\xfb\xe2\xe2\xb7\x8d\xf8\x90\x30\x8b\x03\x12\x95\xa4\x5a\xea\x74\xcc\x1e\xa9\xa6\xde\x1a\x9c\x4c\x8f\x22\x92\xf0\x38\x22\xa6\x75\x5c\xf7\x8e\x42\xd5\x1b\x0e\x93\x1d\x95\xe5\x4f\xdf\x7b\xde\xf9\xd7\x94\x53\xf8\x52\x72\x1f\xe8\xf5\x6f\x43\x02\x51\x7d\x1e\x7a\xe6\xf5\xe0\xab\x6c\x76\x88\x83\xaf\x22\x96\xd3\xbf\x69\xdf\xdd\x11\x81\xaf\xcf\xbf\xb2\x13\xbc\x03\x5c\xba\x4f\x3c\x22\xd8\xd2\x51\xfd\x03\x2d\x7f\x93\x44\xfc\x79\xf9\xe5\x4c\xb3\xac\x3c\xbb\xbc\x68\x4e\xa5\xc4\xec\x31\x7e\xc7\x91\x8d\xd5\x70\xc9\x3c\x5f\xda\x8d\x61\x79\x6a\x7a\xd1\x93\x04\xc6\xba\xce\xa6\x9d\x13\xa7\x2d\x5f\x30\xc3\x1f\xdf\x4d\x14\x9f\xa9\xd7\x6d\x3f\x08\xb5\x17\x98\x0c\xf9\x63\xea\x74\xac\xbd\x77\x2f\x3f\x17\x0b\x80\x9a\xc5\x7f\xf1\xf1\x98\xb1\x93\x1f\x2f\x06\x21\xdf\xc7\xd3\xbd\xf6\xc3\xaf\x6c\xe4\x1f\x87\xa4\x7d\x79\x6d\x41\x4a\x66\x37\xf1\xa1\x30\x7a\x3f\x29\x73\x67\x8b\xfd\xa7\x5d\x99\x1f\xae\x56\xf6\xdf\xe6\x33\xbc\xd5\x69\xf5\x57\x00\x00\x00\xff\xff\x78\x30\xec\x51\x0e\x2b\x00\x00")
```
Comment (thaJeztah) [inline cli/compose/schema/bindata.go]: Should we have a test that validates if this matches the `data/config_schema_v3.0.json` file?

Comment (dnephin) [inline cli/compose/schema/bindata.go]: It should actually already be covered by `hack/validate/compose-bindata`. Maybe it's not working

Comment (dnephin) [inline cli/compose/schema/bindata.go]: Oh, it's missing from `hack/validate/default` I guess that's the problem

Comment (dnephin) [inline cli/compose/schema/bindata.go]: Fixed it in this branch, and #31654 for master

Comment (thaJeztah) [inline cli/compose/schema/bindata.go]: cool, thanks!

**Review by thaJeztah — APPROVED** [review]
LGTM (if green)

ping @vieux please confirm, as this is against the 17.03 branch

**Comment by vieux** [comment]
LGTM thanks

**Pull request merged**